data/map systems.txt

```diff
@@ -12556,6 +12773,39 @@ system "Epsilon Leonis"
 			distance 515
 			period 37.9234
 
+system Equus
+	pos 313.25 -1051.08
+	attributes "neutron" "notable star"
+	arrival 500
+	habitable 355
```

Comment:
```suggestion
	habitable 405
```
Giving neutron-small a habitable range of 50, half that of the regular neutron star.